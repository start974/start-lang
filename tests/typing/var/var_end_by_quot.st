def a := 1
def a' : N := a
def a'' := a'
