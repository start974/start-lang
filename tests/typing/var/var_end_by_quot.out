def a : N := 1
def a' : N := a
def a'' : N := a'
